drivers/adc/ad713x/ad713x.c

```diff
@@ -541,6 +541,10 @@ int32_t ad713x_init(struct ad713x_dev **device,
 
 	dev->dev_id = init_param->dev_id;
 
+	ret = ad713x_spi_reg_read(dev, AD713X_REG_CHIP_TYPE, &data);
```

Comment:
what's the point of reading the chip_type register if not ID check is performed?

Reply:
Good point. Will implement.